server/authorization.go

```diff
@@ -523,7 +523,12 @@ func (a *AuthorizationServer) createEnvoyDenied(req *authv3.CheckRequest, envReq
 
 	// Envoy does not send metadata to ALS on a reject, so we create the
 	// analytics record here and the ALS handler can ignore the metadataless record.
-	if authContext != nil && api != "" {
+	if tracker != nil { // if no tracker, we don't even have org and env context
+		if authContext == nil {
```

Comment:
Can we have a debug log in this block. It's currently not visible so rather difficult to tell.

Reply:
This can only happen when called here, so it's already logged once: https://github.com/apigee/apigee-remote-service-envoy/blob/2f9789b0f70d82c2e0f2d93bd8c15506c624106c/server/authorization.go#L126

Reply:
Oops. Sorry, wrong line. https://github.com/apigee/apigee-remote-service-envoy/blob/2f9789b0f70d82c2e0f2d93bd8c15506c624106c/server/authorization.go#L71

Reply:
I'll add it